app/presenters/step_by_step_page_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class StepByStepPagePresenter
+  include TimeOptionsHelper
+  attr_reader :step_by_step_page
+
+  def initialize(step_by_step_page)
+    @step_by_step_page = step_by_step_page
+  end
+
+  def summary_metadata
+    items = {
+      "Status": step_by_step_page.status[:text],
```

Comment:
I didn't spot this before, but in Ruby the convention is to use hash rocket (`=>`) for string keys.
So `"Status" => step_by_step_page.status[:text]` etc.

Doing that means that the you can access data with out having to call `to_sym` on the key, so things like `items["Status"]` and `items["Last saved"]` will return a value.

Reply:
Good shout!